.github/workflows/ci.yml

```diff
@@ -90,6 +90,12 @@ jobs:
         openai-version: ["1.2.4"]
     steps:
       - uses: actions/checkout@v4
+
+      - name: Create .guardrailsrc
+        run: |
```

Comment:
let's also add this step to examples_check

https://github.com/guardrails-ai/guardrails/blob/7a1a94ac8f8991c7da429497769aad1207c257c4/.github/workflows/examples_check.yml